modules/ROOT/pages/disable-outbound-policies.adoc

```diff
@@ -0,0 +1,77 @@
+= Disable Outbound Policy
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, outbound-policies, resource-level
+
+With Mule 4, you can now apply policies on the outgoing HTTP requests to a Mule runtime engine (Mule) application, in addition to applying policies to the incoming HTTP requests. 
+
+Policies applied to outgoing HTTP requests are known as Outbound policies and are implemented by defining an _http-policy:operation_ block in the policy template, which specifies the execution components.
+
+Mule then applies the operation block of every policy that is applied to an API, to every HTTP requester element defined in a  Mule application.
+
+In the case of the HTTP proxy, there is only one HTTP Requester defined to which an outbound policy is applied.
+
+Because more complex Mule applications that implement APIs may be invoking several external services, more than one HTTP Requesters can be defined for such APIs. In that case, outbound policies are applied to each one of the HTTP requests that the application performs.
```

Comment:
```suggestion
Because Mule applications that implement APIs might invoke several external services, you can define more than one HTTP Requester for such APIs. In that case, outbound policies are applied to each one of the HTTP requests that the application performs.
```

Reply:
Done! Thanks!